.github/actions/apt/provision/action.yml

```diff
@@ -7,8 +7,6 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Update installed packages
-      uses: ./.github/actions/apt/update
```

Comment:
This needs to be kept. Just remove the `upgrade` from the action.

Reply:
You can probably just change the `name` to something else since it doesn't actually `upgrade` packages:

`Update package list` is probably better.

Reply:
After taking it out, the builds still go through succressfully, and I don't see this being called anywhere. Do we really need it?

Reply:
Well, I'm not really sure, but I think it might break if an old version of a package referenced in the package list on the system no longer exists in the repository and it is trying to install it. I'm not sure exactly how Debian/Ubuntu does old package pruning.

Reply:
Here, it really depends on how often `GitHub` updates their `ubuntu-latest`/`ubuntu-20.04` VM images.

Reply:
Newer versions of `apt` might address this issue, but you can leave it as is and perhaps a fix will be needed in the future.

Reply:
Yes, but updating the list but not upgrading will most probably lead to problems of expecting wrong versions. I would go with this right now, and if needed we add fixes.